### PR TITLE
NETOBSERV-656 follow-up: add chart tooltips

### PR DESCRIPTION
**Pull request by jotak**
[NETOBSERV-656](https://issues.redhat.com//browse/NETOBSERV-656)

### CODE REVIEW

File: web/locales/en/plugin__netobserv-plugin.json [hunk 1]
```diff
@@ -126,9 +126,13 @@
   "Unable to get overview": "Unable to get overview",
   "No results found": "No results found",
   "Clear all filters and try again.": "Clear all filters and try again.",
+  "Also show total traffic for the selected filters": "Also show total traffic for the selected filters",
   "Show total": "Show total",
+  "Show other traffic": "Show other traffic",
   "Show others": "Show others",
+  "Show scope-internal traffic, depending on the selected scope (node-internal traffic, namespace-internal traffic)": "Show scope-internal traffic, depending on the selected scope (node-internal traffic, namespace-internal traffic)",
   "Show internal": "Show internal",
+  "Show out of scope traffic, e.g. host-network traffic when scope is not Node.": "Show out of scope traffic, e.g. host-network traffic when scope is not Node.",
```
Comment (jotak) [inline web/locales/en/plugin__netobserv-plugin.json:135]: @memodi what do you think about those texts? Are they good enough, or would you reformulate?

Comment (jotak) [reply web/locales/en/plugin__netobserv-plugin.json]: (I've changed them, cf last commit)

**Comment by jotak** [comment]
I've also added tooltips on donut charts titles:
Capture d’écran du 2022-11-09 10-39-53

Capture d’écran du 2022-11-09 10-39-39

**Review by jpinsonneau — APPROVED** [review]
LGTM in terms of code

Let's wait for more feedback on text contents before merge

**Comment by memodi** [comment]
Thanks for adding tool tips, looks good!

/qe-approved

**Pull request merged**